automation/connect_to_rhel_console.exp

```diff
@@ -6,4 +6,4 @@ set kubeconfig [lindex $argv 0]
 set vm_name [lindex $argv 1]
 
 spawn ./virtctl --kubeconfig=$kubeconfig console $vm_name
-expect default { exit 1 } "*Welcome*\r"
\ No newline at end of file
+expect default { exit 1 } "*Successfully connected to*"
```

Comment:
Please keep this check as it is. We need to check if vm is really up by checking boot log.

Reply:
Which scenario is expected here? VM boot or console connection?

Reply:
It checks here vm boot.